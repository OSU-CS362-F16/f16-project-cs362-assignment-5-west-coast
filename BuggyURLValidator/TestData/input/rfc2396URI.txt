http://www.ietf.org/rfc/rfc2396.txt
http://www.w3.org/Addressing/
ftp://ds.iternic.net/rfc/
http://www.ics.uci.edu/pub/ietf/uri/historical.html#WARNING