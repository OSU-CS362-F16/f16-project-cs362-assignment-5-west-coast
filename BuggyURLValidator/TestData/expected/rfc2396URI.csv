testcase,scheme,ALLOW_LOCAL_URLS,NO_FRAGMENTS,ALLOW_2_SLASHES,ALLOW_ALL_SCHEMES,regex,http://www.ietf.org/rfc/rfc2396.txt,http://www.w3.org/Addressing/,ftp://ds.iternic.net/rfc/,http://www.ics.uci.edu/pub/ietf/uri/historical.html#WARNING
0,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,false,true
1,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true
2,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true
3,{[http][https][ftp]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true
4,{[http][https][ftp]},OFF,OFF,ON,OFF,DEFAULT,true,true,true,true
5,{[http][https][ftp]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true
6,{[http][https][ftp]},OFF,ON,OFF,OFF,DEFAULT,true,true,true,false
7,{[http][https][ftp]},OFF,ON,OFF,ON,DEFAULT,true,true,true,false
8,{[http][https][ftp]},OFF,ON,ON,OFF,DEFAULT,true,true,true,false
9,{[http][https][ftp]},OFF,ON,ON,ON,DEFAULT,true,true,true,false
10,{[http][https][ftp]},ON,OFF,OFF,OFF,DEFAULT,true,true,true,true
11,{[http][https][ftp]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true
12,{[http][https][ftp]},ON,OFF,ON,OFF,DEFAULT,true,true,true,true
13,{[http][https][ftp]},ON,OFF,ON,ON,DEFAULT,true,true,true,true
14,{[http][https][ftp]},ON,ON,OFF,OFF,DEFAULT,true,true,true,false
15,{[http][https][ftp]},ON,ON,OFF,ON,DEFAULT,true,true,true,false
16,{[http][https][ftp]},ON,ON,ON,OFF,DEFAULT,true,true,true,false
17,{[http][https][ftp]},ON,ON,ON,ON,DEFAULT,true,true,true,false
18,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,false,true
19,{[http][https]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true
20,{[http][https]},OFF,OFF,ON,OFF,DEFAULT,true,true,false,true
21,{[http][https]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true
22,{[http][https]},OFF,ON,OFF,OFF,DEFAULT,true,true,false,false
23,{[http][https]},OFF,ON,OFF,ON,DEFAULT,true,true,true,false
24,{[http][https]},OFF,ON,ON,OFF,DEFAULT,true,true,false,false
25,{[http][https]},OFF,ON,ON,ON,DEFAULT,true,true,true,false
26,{[http][https]},ON,OFF,OFF,OFF,DEFAULT,true,true,false,true
27,{[http][https]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true
28,{[http][https]},ON,OFF,ON,OFF,DEFAULT,true,true,false,true
29,{[http][https]},ON,OFF,ON,ON,DEFAULT,true,true,true,true
30,{[http][https]},ON,ON,OFF,OFF,DEFAULT,true,true,false,false
31,{[http][https]},ON,ON,OFF,ON,DEFAULT,true,true,true,false
32,{[http][https]},ON,ON,ON,OFF,DEFAULT,true,true,false,false
33,{[http][https]},ON,ON,ON,ON,DEFAULT,true,true,true,false
34,{[http][https][ftp]},OFF,OFF,OFF,OFF,[a-z],true,true,true,true
35,{[http][https][ftp]},OFF,OFF,OFF,ON,[a-z],true,true,true,true
36,{[http][https][ftp]},OFF,OFF,ON,OFF,[a-z],true,true,true,true
37,{[http][https][ftp]},OFF,OFF,ON,ON,[a-z],true,true,true,true
38,{[http][https][ftp]},OFF,ON,OFF,OFF,[a-z],true,true,true,false
39,{[http][https][ftp]},OFF,ON,OFF,ON,[a-z],true,true,true,false
40,{[http][https][ftp]},OFF,ON,ON,OFF,[a-z],true,true,true,false
41,{[http][https][ftp]},OFF,ON,ON,ON,[a-z],true,true,true,false
42,{[http][https][ftp]},ON,OFF,OFF,OFF,[a-z],true,true,true,true
43,{[http][https][ftp]},ON,OFF,OFF,ON,[a-z],true,true,true,true
44,{[http][https][ftp]},ON,OFF,ON,OFF,[a-z],true,true,true,true
45,{[http][https][ftp]},ON,OFF,ON,ON,[a-z],true,true,true,true
46,{[http][https][ftp]},ON,ON,OFF,OFF,[a-z],true,true,true,false
47,{[http][https][ftp]},ON,ON,OFF,ON,[a-z],true,true,true,false
48,{[http][https][ftp]},ON,ON,ON,OFF,[a-z],true,true,true,false
49,{[http][https][ftp]},ON,ON,ON,ON,[a-z],true,true,true,false
50,{[http][https]},OFF,OFF,OFF,OFF,[a-z],true,true,false,true
51,{[http][https]},OFF,OFF,OFF,ON,[a-z],true,true,true,true
52,{[http][https]},OFF,OFF,ON,OFF,[a-z],true,true,false,true
53,{[http][https]},OFF,OFF,ON,ON,[a-z],true,true,true,true
54,{[http][https]},OFF,ON,OFF,OFF,[a-z],true,true,false,false
55,{[http][https]},OFF,ON,OFF,ON,[a-z],true,true,true,false
56,{[http][https]},OFF,ON,ON,OFF,[a-z],true,true,false,false
57,{[http][https]},OFF,ON,ON,ON,[a-z],true,true,true,false
58,{[http][https]},ON,OFF,OFF,OFF,[a-z],true,true,false,true
59,{[http][https]},ON,OFF,OFF,ON,[a-z],true,true,true,true
60,{[http][https]},ON,OFF,ON,OFF,[a-z],true,true,false,true
61,{[http][https]},ON,OFF,ON,ON,[a-z],true,true,true,true
62,{[http][https]},ON,ON,OFF,OFF,[a-z],true,true,false,false
63,{[http][https]},ON,ON,OFF,ON,[a-z],true,true,true,false
64,{[http][https]},ON,ON,ON,OFF,[a-z],true,true,false,false
65,{[http][https]},ON,ON,ON,ON,[a-z],true,true,true,false
66,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true
